libs/balances-and-allowances/src/const.ts

```diff
@@ -0,0 +1,4 @@
+import { Erc20Abi, Erc20Interface } from '@cowprotocol/abis'
+import { Interface } from '@ethersproject/abi'
+
+export const erc20Interface = new Interface(Erc20Abi) as Erc20Interface
```

Comment:
should tnis interface be exported from '@cowprotocol/abis' (in case some other lib needs it too?)

Reply:
Great suggestion! Fixed